openedx_external_enrollments/tests/external_enrollments/tests_salesforce_external_enrollment.py

```diff
@@ -230,12 +239,17 @@ def test_get_courses_data(self, get_course_mock, get_date_mock, get_salesforce_m
         course_mock.display_name = 'test-course'
         course_mock.other_course_settings = {'salesforce_data': {}}
         get_course_mock.return_value = course_mock
-        get_date_mock.return_value = now
+        course_key_mock = Mock()
+        course_key_mock.org = 'PX'
+        course_key_mock.course = "test-course-run-id"
```

Comment:
I think you can use CourseKey.from_string('course-string'), to obtain an actual CourseKey instance instead of mocking it. @ivanvgh

Reply:
Your are right, applied.